src/control/PdfCache.cpp

```diff
@@ -52,9 +48,7 @@ void PdfCache::setRefreshThreshold(double threshold) { this->zoomRefreshThreshol
 void PdfCache::setAnyZoomChangeCausesRecache(bool b) { this->zoomClearsCache = b; }
 
 void PdfCache::clearCache() {
-    for (PdfCacheEntry* e: this->data) {
-        delete e;
-    }
+    for (PdfCacheEntry* e: this->data) { delete e; }
```

Comment:
This is not an intentional change. My editor changed it, but I was not sure if it should had been like this. What is the command to format the project according to the clang style? I assumed my editor was using the right clang format settings because it asked me for the project root directory.

Reply:
@createyourpersonalaccount I've heard of clang format but havn't had the chance to use it yet and this "review" is my first interaction with this project outside of using the software as an end user. Someone else will have to answer that question. Sorry!

Reply:
The formatting change looks correct to me. The pipeline clang format check would complain otherwise.

Reply:
Please revert this change.

Reply:
@jotoho is correct in his assessment, that it should not include changes where you only modified the format. Whatever your IDE did there should not have been done.